lib/os/assert.c

```diff
@@ -53,3 +54,4 @@ void assert_print(const char *fmt, ...)
 
 	va_end(ap);
 }
+EXPORT_SYMBOL(assert_print);
```

Comment:
Again, these are global symbols - there is no implicit dependency on llext for assert functionality.

I can understand the case where a particular private or static function needs to be exported for llext (for visibility), but for globals, standard C functions, can we just put them into `llext_export.c` as well?